src/e2e/app/cli/repo-tests/src/rest_api/test_gql_query.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use kamu_cli_e2e_common::{KamuApiServerClient, KamuApiServerClientExt};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_gql_query_mut_create_empty_returns_correct_alias_mt(
+    mut kamu_api_server_client: KamuApiServerClient,
+) {
+    let token = kamu_api_server_client.auth().login_as_e2e_user().await;
+    kamu_api_server_client.set_token(Some(token));
```

Comment:
`login_as_e2e_user()` method escapes the token installation logic, so the `token` does not need to be set explicitly

Reply:
done